dirs/dirs.go

```diff
@@ -25,17 +25,18 @@ import "path/filepath"
 var (
 	GlobalRootDir string
 
-	SnapAppsDir      string
-	SnapOemDir       string
-	SnapDataDir      string
-	SnapDataHomeGlob string
-	SnapAppArmorDir  string
-	SnapSeccompDir   string
-	SnapUdevRulesDir string
-	LocaleDir        string
-	SnapIconsDir     string
-	SnapMetaDir      string
-	SnapLockFile     string
+	SnapAppsDir               string
+	SnapOemDir                string
+	SnapDataDir               string
+	SnapDataHomeGlob          string
+	SnapAppArmorDir           string
+	SnapAppArmorAdditionalDir string
```

Comment:
Good point, the description is very generic. The hardware assign rules (or exceptions) are stored there right now. Its a relatively generic mechanism however its used only for hw assign right now (and I presume for capabilities later). I'm happy to rename to "SnapAppArmorHwAssignRulesDir" or SnapAppArmorCapabilitiesRulesDir" if you prefer that name.